apps/examples/binary_update/Kconfig

```diff
@@ -7,9 +7,18 @@ config EXAMPLES_BINARY_UPDATE
 	bool "Binary Update example"
 	default n
 	depends on BINMGR_UPDATE
+	select FOTA_DELTA_ENGINE
 	---help---
 		Enable Binary Update example
 
+if EXAMPLES_BINARY_UPDATE
+config FOTA_DELTA_UPDATE
```

Comment:
Please add the prefix to know that it is a config for example.
How about `EXAMPLES_BINARY_DELTA_UPDATE`?
Because here is no FOTA functionality.